src/zino/tasks/linkstatetask.py

```diff
@@ -135,18 +135,20 @@ def _make_or_update_state_event(self, port: Port, new_state: InterfaceState, las
         event.add_log(log)
         self.state.events.commit(event)
 
-        self._schedule_verification_of_single_port(port.ifindex)
-
-    def _schedule_verification_of_single_port(self, ifindex: int):
-        in_two_minutes = datetime.datetime.now() + datetime.timedelta(minutes=2)
-        job_name = f"{self.device.name}-verify-{ifindex}-state"
+        self.schedule_verification_of_single_port(port.ifindex)
+
+    def schedule_verification_of_single_port(
+        self, ifindex: int, deadline: datetime.timedelta = datetime.timedelta(minutes=2), reason: str = "verify"
+    ):
+        """Schedules a verification of a single port at a given time in the future"""
+        when = datetime.datetime.now() + deadline
```

Comment:
I do not like the variable name `when` - what about something like `verification_time`?

Reply:
Done :)